src/components/ui/VideoQuestion/VideoQuestion.jsx

```diff
@@ -0,0 +1,20 @@
+import YouTubePlayer from "./YouTubePlayer";
```

Comment:
Why arent we using a library for this and reinvint the wheel? The reason why we dont reinvint the wheel is because there a lot of edge cases we havent covered, there can be a lot of bugs along the way. So we use a battle tested library that have fixed these issues.
For example this:
https://www.npmjs.com/package/react-youtube